### ISSUE TITLE
Запоминать выбранные каналы для последующих публикаций

### ISSUE THREAD

**Issue by nikolaev-rd** [Telepost-me/support#46]
**Краткое описание**
Сейчас при создании поста и выборе каналов, где будет опубликован пост, этот выбор не запоминается. Приходится выбирать каналы каждый раз при создании постов.

**Как вы видите решение/реализацию**
После создания поста есть возможность указать, чтобы Телепост запомнил выбор каналов из этого поста.

**Альтернативные варианты**
Можно еще сделать, чтобы выбор запоминался хотя бы для черновика, и сохранялся для всех копий постов из этого черновика (типа шаблон).

**Дополнительно**
Ссылка на обсуждение: https://t.me/c/1306965784/27336

**Comment by telepost-support** [comment]
Нужно ли запоминать каналы если ты зашел в канал `А`, при создании поста выбрал каналы `В` `С`. Следующий раз когда ты в канале `А` создаешь пост, выбраны каналы `В` `С`? 
Не очень логично так как ты в канале `А` и он должен быть выбран по умолчанию при создании поста.

**Comment by nikolaev-rd** [comment]
> Нужно ли запоминать каналы если ты зашел в канал `А`, при создании поста выбрал каналы `В` `С`. Следующий раз когда ты в канале `А` создаешь пост, выбраны каналы `В` `С`?
> Не очень логично так как ты в канале `А` и он должен быть выбран по умолчанию при создании поста.

Согласен. Мне альтернативный вариант с шаблоном (пост в черновиках с сохраненной настройкой в какие каналы постить) кажется более удобным и логичным.
